src/errorsMessage.js

```diff
@@ -0,0 +1,11 @@
+const { ERROR_MESSAGE, CASE_OPTIONS } = require('./constants/constants');
+
+const errorMessages = {
+  NO_TEXT: 'Text to convert is required. '
+    + `Correct request is: ${ERROR_MESSAGE}`,
```

Comment:
This part `Correct request is: ` also is repeating twice, it can be moved to the same constant